src/ImageSharp/PixelFormats/Generated/PixelOperations{TPixel}.Generated.cs

```diff
@@ -20,8 +21,8 @@ internal virtual void PackFromRgba32(Span<Rgba32> source, Span<TPixel> destPixel
         {
 			GuardSpans(source, nameof(source), destPixels, nameof(destPixels), count);
             
-            ref Rgba32 sourceRef = ref source.DangerousGetPinnableReference();
-            ref TPixel destRef = ref destPixels.DangerousGetPinnableReference();
+            ref Rgba32 sourceRef = ref MemoryMarshal.GetReference(source);
```

Comment:
We also need to apply these changes to the `.tt` files. This code is auto-generated.

Reply:
Got stuck trying deterministically reference a nuget package in the T4 template. 

something like...  <#@ assembly name="$(NuGetPackageRoot)/system.memory/4.5.0-preview1-26216-02/lib/netstandard2.0/System.Memory.dll" #>

Will take a fresh stab tomorrow.

Reply:
@iamcarbon you don't need that! T4 is just a dumb tool for generating the code, it doesn't need to know what the generated code does or references. Actually you can generate any text with it. The syntax is almost the same as classic ASP.

Reply:
@antonfirsov ahhhhh! was overthinking this for no reason. updated.